tutorial_2.md

```diff
@@ -1,41 +1,69 @@
+
+![API Token Creation](screenshots/api_token.png)
+*Click "Create token" in the Access tokens section*
+
+When configuring your token, give it a descriptive name like "multi-agent-chatbot", select **"Writer"** as the role (required for creating configurations), use the default API version (latest), and leave "This is a service token" unchecked for now.
+
+![Name API Token](screenshots/name_api_token.png)
+*Configure your token with a descriptive name and Writer role*
+
+After configuring the settings, click **"Save token"** and immediately copy the token value. This is **IMPORTANT** because it's only shown once!
+
+![Copy API Token](screenshots/copy_api_token.png)
+*Copy the token value immediately - it's only shown once*
+
+Finally, add the token to your environment:
+   ```bash
+   # Add this line to your .env file
+   LD_API_KEY=your-copied-api-key-here
+   ```
+
+**Security Note**: Keep your API key private and never commit it to version control. The token allows full access to your LaunchDarkly account.
+
+## Step 1: Add External Research Tools (4 minutes)
+
+Your agents need more than just your internal documents. **Model Context Protocol (MCP)** connects AI assistants to live external data and they agents become orchestrators of your digital infrastructure, tapping into databases, communication tools, development platforms, and any system that matters to your business. MCP tools run as separate servers that your agents call when needed.
+
+The [MCP Registry](https://github.com/modelcontextprotocol/registry) serves as a community-driven directory for discovering available MCP servers - like an app store for MCP tools. Browse available servers at [registry.modelcontextprotocol.io](https://registry.modelcontextprotocol.io/docs#/operations/list-servers) to see what's currently available. Registry servers install through standard package managers:
 
 ```bash
 # Example registry installations (not used in this tutorial)
-pip install reddit-research-mcp    # Reddit research with citations  
-npx scorecard-ai-mcp               # LLM evaluation tools
+npx @modelcontextprotocol/server-filesystem    # File system access
+npx @modelcontextprotocol/server-slack         # Slack workspace integration  
+pip install mcp-server-git                     # Git repository integration
 ```
 
-**Our Approach:** For this tutorial, we'll use manual installation since our specific academic research servers (ArXiv and Semantic Scholar) aren't yet available in the registry.
+For this tutorial, we'll use manual installation since our specific academic research servers (ArXiv and Semantic Scholar) aren't yet available in the registry.
 
 Install external research capabilities:
 
```

Comment:
I'm getting a warning that the arxiv server is already installed, so this might not be necessary if folks have followed the previous tutorial.

Reply:
Do I need to do anything to install semanticscholar, other than cloning it?